### PR TITLE
Adds text that appears after cloning/replica pod cloning

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Adds text that appears after cloning/replica pod cloning

## Why It's Good For The Game
Its kind of an unwritten rule that you forget some of your life that happened if you get cloned. But player confusion still happens. This PR aims to rectify that by directly saying it to the player that got cloned.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Medbay cloning text : 
image

</details>

## Changelog
:cl:
add: Added text that appears to the person got cloned, stating they forgot some of their previous life
config: Added a config option to show text that appears to the person got cloned, stating they forgot five minutes of their previous life
spellcheck: Fixed a typo in a comment about the polymorph staff policy
/:cl:

### CODE REVIEW

**Comment by KnighTheThrasher** [comment]
Rule 14 says that when you get cloned, you forget 5 minutes leading up to your death. Not 10 minutes.

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -515,6 +515,7 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+		to_chat(occupant, "<span class='boldannounce'>You have forgotten all the knowledge you gained while being a ghost aswell as the ten minutes leading up to your death!</span>")
```
Comment (CydiaLamiales) [inline code/game/machinery/cloning.dm]:
```suggestion
		to_chat(occupant, "<span class='boldannounce'>You have forgotten all the knowledge you gained while being a ghost aswell as the five minutes leading up to your death!</span>")
```

File: code/modules/hydroponics/grown/replicapod.dm [hunk 2]
```diff
@@ -156,6 +156,7 @@
 	podman.set_cloned_appearance()
 	// On harvest
 	to_chat(podman, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+	to_chat(podman, "<span class='boldannounce'>You have forgotten all the knowledge you gained while being a ghost aswell as everything that happened after you had the blood sample taken from you!</span>")
```
Comment (CydiaLamiales) [inline code/modules/hydroponics/grown/replicapod.dm]:
```suggestion
	to_chat(podman, "<span class='boldannounce'>You have forgotten all the knowledge you gained while being a ghost aswell as the five minutes leading up to your death!</span>")
```

**Review by CydiaLamiales — COMMENTED** [review]
Forgot

**Comment by EvilDragonfiend** [comment]
It's uo to a server policy. You would need a server cfg.

File: code/game/machinery/cloning.dm [hunk 3]
```diff
@@ -515,6 +515,7 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+		to_chat(occupant, "<span class='boldannounce'>You have forgotten all the knowledge you gained while being a ghost aswell as the five minutes leading up to your death!</span>")
```
Comment (EvilDragonfiend) [inline code/game/machinery/cloning.dm]:
```suggestion
		if(!experimental && !experimental_pod)
			to_chat(occupant, "<span class='boldannounce'>You have forgotten all the knowledge you gained while being a ghost aswell as the five minutes leading up to your death!</span>")
```

You shouldn't show this message to exp clones.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
oh shit it's in other proc... hang on.

File: code/game/machinery/cloning.dm [hunk 4]
```diff
@@ -515,6 +515,8 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+		if(!experimental && !experimental_pod)
```
Comment (EvilDragonfiend) [inline code/game/machinery/cloning.dm]:
```suggestion
		if(!experimental_pod && !connected?.experimental) // connected computer can be null, so put '?' here. Let's not make this message to be seen by experimental clones.
```

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Review by Penwin0 — DISMISSED** [review]

**Review by itsmeow — CHANGES_REQUESTED** [review]
should be a config for downstreams

**Comment by CydiaLamiales** [comment]
There, added that

**Comment by itsmeow** [comment]
it should probably be a string type config rather than a hardcoded flag

**Comment by CydiaLamiales** [comment]
> it should probably be a string type config rather than a hardcoded flag

I tried but couldn't get it to read the text, gonna need help with that.

**Comment by CydiaLamiales** [comment]
There

**Review by itsmeow — DISMISSED** [review]

**Comment by itsmeow** [comment]
@Crossedfall

File: code/__DEFINES/admin.dm [hunk 5]
```diff
@@ -102,7 +102,7 @@
 #define STICKYBAN_ROGUE_CHECK_TIME 5
 
 
-#define POLICY_POLYMORPH "polymorph" //Shown to vicitm of staff of change and related effects.
+#define POLICY_POLYMORPH "policy_polymorph" //Shown to vicitm of staff of change and related effects.
```
Comment (Crossedfall) [inline code/__DEFINES/admin.dm]: Undocumented/unrelated change?

Comment (CydiaLamiales) [reply code/__DEFINES/admin.dm]: Oh that was from me testing stuff, yeah unrelated.

Comment (CydiaLamiales) [inline code/__DEFINES/admin.dm]:
```suggestion
#define POLICY_POLYMORPH "polymorph" //Shown to victim of staff of change and related effects.
```

Comment (itsmeow) [reply code/__DEFINES/admin.dm]: No just remove them

Comment (itsmeow) [reply code/__DEFINES/admin.dm]: They're unused

Comment (CydiaLamiales) [reply code/__DEFINES/admin.dm]: There, removed.

**Review by Crossedfall — APPROVED** [review]
